Doc/library/os.rst

```diff
@@ -1087,9 +1087,9 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
    .. versionadded:: 3.3
 
 
-.. function:: pwrite(fd, str, offset)
+.. function:: pwrite(fd, buffer, offset)
 
-   Write *bytestring* to a file descriptor, *fd*, from *offset*,
+   Write *buffer* to a file descriptor, *fd*, from *offset*,
    leaving the file offset unchanged.
 
```

Comment:
Can you please document that the function returns the number of written bytes?

Example from os.write doc: "Return the number of bytes actually written."

Reply:
Fair enough.